glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -556,6 +557,25 @@ open class GleanInternalAPI internal constructor () {
     internal fun testHasPingType(pingName: String): Boolean {
         return LibGleanFFI.INSTANCE.glean_test_has_ping_type(handle, pingName).toBoolean()
     }
+
+    @VisibleForTesting
+    internal var isMainProcess: Boolean? = null
```

Comment:
Why is this `internal`? Can't this just be `private`?

Reply:
Because of the test that I hadn't copied over from AC yet.